tests/waku_rln_relay/test_wakunode_rln_relay.nim

```diff
@@ -595,4 +595,4 @@ procSuite "WakuNode - RLN relay":
       await node.mountRlnRelay(wakuRlnConfig)
     except CatchableError as e:
       check e.msg ==
-        "failed to mount WakuRlnRelay: relay message limit (rln) cannot be exceed 100"
+        "failed to mount WakuRlnRelay: relay-message-limit can't be exceed then user-message-limit set by contract"
```

Comment:
this file uses the static mode of operation, so there is no point checking the user message limit from the contract

Reply:
This test should actually go in the `tests/waku_rln_relay/test_rln_group_manager_onchain.nim` file